Command/Validators.php

```diff
@@ -50,7 +50,10 @@ public static function validateEntityName($shortcut)
         $entity = str_replace('/', '\\', $shortcut);
 
         if (false === $pos = strpos($entity, ':')) {
-            throw new \InvalidArgumentException(sprintf('The entity name must contain a : ("%s" given, expecting something like AcmeBlogBundle:Post)', $entity));
+            throw new \InvalidArgumentException(sprintf(
+                'The entity name must contain a : ("%s" given, expecting something like AcmeBlogBundle:Post)',
```

Comment:
While you're at it : ":" => ": sign" or "colon sign", otherwise the error message is a bit hard to understand